src/screen/dashboard/command_bar.rs

```diff
@@ -125,12 +132,53 @@ impl std::fmt::Display for Command {
     }
 }
 
+impl Buffer {
+    fn list(clients: &data::client::Map) -> Vec<Self> {
+        let mut channels = vec![];
+
+        for (server, state) in clients.iter() {
+            match state {
+                data::client::State::Ready(connection) => {
+                    for channel in connection.channels() {
+                        channels.push(data::Buffer::Channel(server.clone(), channel.clone()));
+                    }
+                }
+                data::client::State::Disconnected => {}
+            }
+        }
+
+        let mut buffers = vec![Buffer::Maximize, Buffer::New, Buffer::Close];
+        buffers.extend(channels.iter().cloned().map(Buffer::Replace));
+        buffers
+    }
+}
```

Comment:
Now that #180 is merged, there is a helper method in `src/dashboard` for getting all buffer entries, `all_buffers`. I'd rather we use that and pass in the list of buffers here so this doesn't depend on anything complex. 

Also, it includes the server & query buffer types which we probably want to include in command bar for switching

Reply:
Note, we may want to remove the `open_buffers` from `all_buffers` to generate a `closed_buffers` list to populate command bar. We shouldn't allow opening the same target in multiple buffers.

In the future, we can maybe add a `switch with <buffer>` command which handles already open buffers.

Reply:
Done with `all_buffers`.

Also removed buffer commands when no buffer is focused, except `New buffer`